### PR TITLE
feat: adding format [mime-type] + upgrade mc-models + types (MAPCO-3257)

### PR DESCRIPTION
**Pull request by ronenkapelian**
Implement Overseer to support and provide Format (Mime) for image on records

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                       |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✖                                                                       |

* Added new attribiurte called format
format is string posiible values (image/png | image/jpeg).
* field calculated by source data and reflected from lookup provided from types package
* upgrade CI node versions (18-20)

### CODE REVIEW

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 16
```
Comment (asafMasa) [inline .github/workflows/pull_request.yaml:17]: docker file is set to 16 please update it to ts-server boiler plate version (20),  same as with the workflow file

File: src/layers/models/layersManager.ts [hunk 2]
```diff
@@ -196,9 +199,10 @@ export class LayersManager {
           this.useNewTargetFlagInUpdateTasks
         );
 
-        const message = `Update job - Transparency and TileOutputFormat will be override from catalog:
+        const message = `Update job - Transparency, TileOutputFormat and TilesMimeFormat will be override from catalog:
```
Comment (asafMasa) [inline src/layers/models/layersManager.ts:202]: Send the "tileMimeFormat" to mapproxy-api instead of "tileOutputFormat"

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good Job!

**Comment by ronenkapelian** [comment]
deprecated

**Pull request closed without merging**